linera-execution/src/transaction_tracker.rs

```diff
@@ -160,6 +164,62 @@ impl TransactionTracker {
         self.operation_result = result
     }
 
+    pub fn add_stream_to_process(
```

Comment:
Maybe just `add_new_event_stream`?

Reply:
But this isn't necessarily a new stream: It can be called either because of a new subscription to a stream, or because of new events in an existing subscription's stream.